### PR TITLE
Fixed Button Name add and get to Create and Join

### PR DESCRIPTION
**Pull request by shikhar13012001**
changed the button name

### CODE REVIEW

**Comment by shikhar13012001** [comment]
Hello sir, I have added labels to the input fields as per direction

**Comment by tstelzle** [comment]
I wanted to actually to see the label "Name" so that the user sees what kind of input he has to give. Something like that 
          `<label>Name</label>`
         ` <label>Game ID</label>`

Furthermore you have reformated the whole file, because you are using 2 tabs. I will not merge it when you change the style. Please open a ticket, with a valid reason why you want to change it to two tabs.

**Comment by shikhar13012001** [comment]
Kind sir, I am extremely sorry for this mistake of label and reformatting, I will change it as it was before, I think there were some extensions in vs which auto formatted it.

**Comment by shikhar13012001** [comment]
Hi again, I have made the requested changes please verify them. I am sorry again for the earlier comment.
Thank you.

File: client/vue/index.html [hunk 1]
```diff
@@ -1,212 +1,161 @@
...
-          </label>
-          <button class="button" v-on:click="create_game">Create Game</button>
+          <label>Name</label>
+            <input v-model="game_name">
+            <button class="button" v-on:click="create_game">Create Game</button>
         </div>
         <div class="box">
-          <label for="Game ID"><input v-model="game_uid" /> </label>
-          <button class="button" v-on:click="get_game">Join Game</button>
+          <label for="Game_ID">Game ID</label>
```
Comment (tstelzle) [inline client/vue/index.html:72]: Please remove `for="Game ID"`as it is not needed here!

File: client/vue/index.html [hunk 2]
```diff
@@ -1,212 +1,161 @@
...
         <div class="box">
-          <h2>Setup</h2>
+            <h2>Setup</h2>
         </div>
         <div class="box">
-          <label for="Name">
-            <input v-model="game_name" />
-          </label>
-          <button class="button" v-on:click="create_game">Create Game</button>
+          <label>Name</label>
```
Comment (tstelzle) [inline client/vue/index.html:67]: This looks good!

**Review by tstelzle — COMMENTED** [review]
Fix the last unnecessary item i mentioned in the review, then I will merge it.

**Pull request merged**